test/suites/clustering.sh

```diff
@@ -10,7 +10,7 @@ test_clustering_enable() {
     # shellcheck disable=SC2034,SC2030
     LXD_DIR=${LXD_INIT_DIR}
 
-    lxc config show | grep "core.https_address" | grep -qE "127.0.0.1:[0-9]{4,5}$"
+    lxc config show | grep -xE "^\s+core\.https_address: 127\.0\.0\.1:[0-9]{4,5}"
```

Comment:
"simplify" :)

Reply:
How come?

Reply:
I stuck with `lxc config show` as this is the only place where it's output is verified:

```
$ git grep 'lxc config show | '
test/suites/clustering.sh:    lxc config show | grep -xE "^\s+core\.https_address: 127\.0\.0\.1:[0-9]{4,5}"
```

It is also the only place where the test suite call `lxc config show` to get the global server config and not an instance's config. So it came down to a question of code coverage.

Reply:
makes sense thanks